lib/main.dart

```diff
@@ -1,113 +1,128 @@
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+        debugShowCheckedModeBanner: false,
+        title: 'Business Card',
+        home: Scaffold(
+          body: Container(
+            decoration: BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage("assets/images/cardbackground.png"),
+                    fit: BoxFit.fill)),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Container(
+                  decoration: BoxDecoration(
+                    shape: BoxShape.circle,
+                    boxShadow: [
+                      BoxShadow(
+                          blurRadius: 35,
+                          color: Constants.avatarShadowColor,
+                          spreadRadius: 1)
+                    ],
+                  ),
+                  child: CircleAvatar(
+                    backgroundImage: AssetImage('assets/images/me.jpg'),
+                    radius: 100,
+                  ),
+                ),
+                SizedBox(
+                  height: 45.0,
+                ),
+                Text('Kevin Guillen',
+                    style: TextStyle(
+                      fontSize: Constants.personNameFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.personNameTextColor,
+                      letterSpacing: 1.0,
+                    )),
+                SizedBox(
+                  height: 10.0,
+                ),
+                Text(
+                  'Flutter Developer Trainee',
+                  style: TextStyle(
+                    fontSize: 20.0,
+                    fontWeight: FontWeight.bold,
+                    color: Constants.positionTextColor,
+                    letterSpacing: 1.0,
+                  ),
+                ),
+                SizedBox(
+                  height: 80.0,
+                  width: 300.0,
+                  child: Divider(
+                    thickness: 2.0,
+                    color: Colors.blueAccent.shade100,
+                  ),
+                ),
```

Comment:
You can remove the SizedBox and follow the example of this page https://api.flutter.dev/flutter/material/Divider-class.html, to use the full configuration of the Divider widget